app/views/manage_assignments/coverages/_coverage.html.erb

```diff
@@ -13,6 +13,9 @@
     </h2>
     <p class="class-card-heading-outcomes">
       <%= t(".matched_outcomes", count: coverage.outcome_coverages.size) %>
+    <p>
```

Comment:
Should this only show if there are no attachments already? If there's already an attachment, should we just show a link to it?

Or should this always be a link and we can link to the existing attachments in the form?